drivers/vcluster/vcluster.go

```diff
@@ -71,7 +71,10 @@ type FIOOptions struct {
 
 // NewVCluster Creates instance of Vcluster
 func NewVCluster(name string) *VCluster {
-	namespace := "vcluster-" + name
+	err := SetDefaultStorageClass()
+	log.FailOnError(err, "Cannot set a default storage class. Exiting the test case.")
+	namespace := fmt.Sprintf("ns-%v-%v", name, time.Now().Unix())
+	log.Infof("Namespace for Vcluster %v is : %v", name, namespace)
 	return &VCluster{Namespace: namespace, Name: name}
```

Comment:
Can we return error here instead of handling it inside the function?

Reply:
Makes sense, Looks like I missed it here - thanks for pointing this out. Done now.